src/igr-tests/environ2/run-test.sh

```diff
@@ -1,41 +1,43 @@
 #!/bin/sh
 
-cd "$(dirname "$0")"
+# FIXME: UID (and possibly GID) cannot unsent on MacOS! Skipping...
```

Comment:
`$UID` is readonly variable on MacOS. We can just ignore this and don't unseting `$UID`?

Reply:
Yes just ignore it